tests/db_functions/models.py

```diff
@@ -59,3 +59,8 @@ class FloatModel(models.Model):
 
 class UUIDModel(models.Model):
     uuid = models.UUIDField(null=True)
+
+
+class UUIDDurationModel(models.Model):
```

Comment:
I'd probably just add `shift` to the existing model, unless you think that makes the test harder to grok?